tests/unit/services/test_logging.py

```diff
@@ -17,3 +20,25 @@ def test_timestamper_without_debug(monkeypatch):
     monkeypatch.setattr("services.logging.DEBUG", False)
 
     assert timestamper(None, None, {"event": "derp"}) == {"event": "derp"}
+
+
+def test_missing_variable_error_filter():
```

Comment:
These tests are really nice. I tried changing the `django.template` logging settings because I thought it might be possible to use some better options and I broke one of the tests as a result, which immediately told me I'd done the wrong thing.